File: solutions/beecrowd/1048/1048.cpp
#include <cstdio>

int main() {
    float n;

    while (scanf("%f", &n) != EOF) {
        if (n <= 400.0) {
            printf("Novo salario: %.2f\n", n * 1.15);
            printf("Reajuste ganho: %.2f\n", n * 1.15 - n);
            puts("Em percentual: 15 %");
        } else if (n <= 800.0) {
            printf("Novo salario: %.2f\n", n * 1.12);
            printf("Reajuste ganho: %.2f\n", n * 1.12 - n);
            puts("Em percentual: 12 %");
        } else if (n <= 1200.0) {
            printf("Novo salario: %.2f\n", n * 1.10);
            printf("Reajuste ganho: %.2f\n", n * 1.10 - n);
            puts("Em percentual: 10 %");
        } else if (n <= 2000.0) {
            printf("Novo salario: %.2f\n", n * 1.07);
            printf("Reajuste ganho: %.2f\n", n * 1.07 - n);
            puts("Em percentual: 7 %");
        } else {
            printf("Novo salario: %.2f\n", n * 1.04);
            printf("Reajuste ganho: %.2f\n", n * 1.04 - n);
            puts("Em percentual: 4 %");
        }
    }

    return 0;
}

File: solutions/beecrowd/1048/1048.py
import sys

for line in sys.stdin:
    salary = float(line)

    if salary <= 400.00:
        print(
            f'Novo salario: {salary * 1.15:.2f}\n'
            f'Reajuste ganho: {salary * 1.15 - salary:.2f}\n'
            'Em percentual: 15 %'
        )
    elif salary <= 800.00:
        print(
            f'Novo salario: {salary * 1.12:.2f}\n'
            f'Reajuste ganho: {salary * 1.12 - salary:.2f}\n'
            'Em percentual: 12 %'
        )
    elif salary <= 1200.00:
        print(
            f'Novo salario: {salary * 1.1:.2f}\n'
            f'Reajuste ganho: {salary * 1.1 - salary:.2f}\n'
            'Em percentual: 10 %'
        )
    elif salary <= 2000.00:
        print(
            f'Novo salario: {salary * 1.07:.2f}\n'
            f'Reajuste ganho: {salary * 1.07 - salary:.2f}\n'
            'Em percentual: 7 %'
        )
    else:
        print(
            f'Novo salario: {salary * 1.04:.2f}\n'
            f'Reajuste ganho: {salary * 1.04 - salary:.2f}\n'
            'Em percentual: 4 %'
        )
